### DISCUSSION TITLE
如果从命令行调用跳转到浏览器打开对应的pdf翻译

### DISCUSSION THREAD

**Discussion by AlexShyXie** [immersive-translate/immersive-translate#1145] [Help]
### 概述 | Summary

请问，如果我想用其他语言从命令行调用跳转到浏览器打开相应的pdf呢，比如我download里有个pdf，我想运行代码:
immersive-translate xxx.pdf
这样直接跳转到浏览器打开pdf翻译

**Comment by theowenyoung** [comment]
首先在扩展管理里，允许沉浸式翻译访问本地文件：

![fkNwyCYneT](https://github.com/immersive-translate/immersive-translate/assets/62473795/c46d0d68-eb3f-4403-b114-ca46fe3aaedf)

然后调用该命令：


```
open -a "Google Chrome" chrome-extension://bpoadfkcbjbfhfodiogcnhhhpibjhbnh/pdf/index.html?file=file%3A%2F%2F%2FUsers%2Fgreen%2FLibrary%2FCloudStorage%2FS3-S3%2Fowen-documents%2Fpdf%2Fapplsci-13-04574.pdf
```

**Reply by AlexShyXie** [reply]
您好，我用windows没有找到open -a "Google Chrome" chrome-extension，您这个是什么命令的？我使用Edge浏览器，用cmd命令运行：
start msedge https://app.immersivetranslate.com/pdf/index.html?file=file%45%3A%2F%2FDownloads%2Ftest.pdf
打开E://downloads/test.pdf，但是直接黑屏
![image](https://github.com/immersive-translate/immersive-translate/assets/49368033/6154dbc4-ac53-4649-a619-122544d1b49d)

我已经设置了允许url：
![image](https://github.com/immersive-translate/immersive-translate/assets/49368033/a37ca5d4-aee4-4297-a702-e8c5fa44907a)

**Reply by theowenyoung** [reply]
请使用插件本地的pdf 页面，而不是线上的地址，线上的地址没有权限读取本地文件：

类似这样：

extension://amkbmndfnliijdhojkpoglbnaaahippg/pdf/index.html?file=https%3A%2F%2Fwww.africau.edu%2Fimages%2Fdefault%2Fsample.pdf

**Reply by AlexShyXie** [reply]
大佬您好，谢谢您的耐心解答，但是即使我换成本地路径仍然存在这样的问题。
我现在用的这个：`start chrome "chrome-extension://bpoadfkcbjbfhfodiogcnhhhpibjhbnh/pdf/index.html?file=file%45%3A%2F%2FDownloads%2Ftest.pdf"`

![image](https://github.com/immersive-translate/immersive-translate/assets/49368033/b7ab2869-0aff-4c62-82ae-36db0428958e)

**Reply by AlexShyXie** [reply]
大佬您好，我搞定了，谢谢
换成了这样，pdf路径不进行url转换，`start chrome "chrome-extension://bpoadfkcbjbfhfodiogcnhhhpibjhbnh/pdf/index.html?file=file:///E:/Downloads/test.pdf"`
